www/tutorial/app/xterm-terminal/xterm-terminal.component.css

```diff
@@ -0,0 +1,24 @@
+#terminal-container {
+    width: 1200px;
+    height: 800px;
+    margin: 0 auto;
+    padding: 2px;
+}
+
+/*the app was having trouble setting the following css
+classes, so i added a host and /deep/ call to ensure they get set*/
+
+:host /deep/ .terminal {
```

Comment:
Lets not worry about this for now. I can have a look, once I am able to run this locally